manifests/init.pp

```diff
@@ -93,6 +96,7 @@
   Array[Stdlib::Absolutepath] $additional_excludes = [],
   Array[Stdlib::Absolutepath] $additional_includes = [],
   String[1] $username                              = $facts['hostname'],
+  Integer[1] $ssh_port                             = 22,
```

Comment:
Can you please use the [Stdlib::Port](https://forge.puppet.com/puppetlabs/stdlib#stdlibport) type?:
```suggestion
  Stdlib::Port $ssh_port                             = 22,
```